### PR TITLE
[Regression] Problem with single quote encode in title event

### PR DESCRIPTION
**Pull request by AtiaZ**

### CODE REVIEW

**Comment by AtiaZ** [comment]
the event data storage is based on the allEvents which is retrieved after calling the getEventInMonth() method (https://github.com/exodev/calendar/blob/ab162abbcb1b2ce35b29244712ce1d96c85820ba/calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarView.java#L603). I have added the evt.setSummary(evt.getSummary().replace("&#39", "'")) when setting the summary in order to a correct result when trying to retrieve the title with single quote.

File: calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIQuickAddEvent.java [hunk 1]
```diff
@@ -297,7 +297,6 @@ public void execute(Event<UIQuickAddEvent> event) throws Exception {
         return;
       }
       summary = summary.trim();
-      summary = CalendarUtils.enCodeTitle(summary);
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIQuickAddEvent.java]: It is laybe not the only location to do this change. I see UIEventForm.java, UITaskForm.java and UICreateEvent.java. Maybe we can even remove the enCodeTitle method.

Comment (AtiaZ) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIQuickAddEvent.java]: ok, i will check.

Comment (AtiaZ) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIQuickAddEvent.java]: done

Comment (thomasdelhomenie) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/popup/UIQuickAddEvent.java]: :+1:

**Pull request merged**